### PR TITLE
[MediaBundle] Fix php 7.4 incompatibility

### LINKED ISSUE #2667: Error on loading DefaultSiteFixtures
When running 

`bin/console doctrine:fixtures:load`

I get error:

`Notice: Trying to access array offset on value of type bool In ImageHandler.php line 96`

I'm using PHP 7.4.3 with installed GD.

See attached screenshots.
error1
error2
testscript
testscript_output
php_modules

### PR DESCRIPTION
**Pull request by acrobat**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | fixes #2667

@igst Can you check if this fixes the error you have?

### CODE REVIEW

**Comment by igst** [comment]
Hi @acrobat,

thanks, yes, its fixed!

File: src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
+use Kunstmaan\MediaBundle\Helper\MimeTypeGuesserFactoryInterface;
+use Kunstmaan\MediaBundle\Helper\ExtensionGuesserFactoryInterface;
+
+class ImageHandlerTest extends TestCase
+{
+    public function testPrepareWithSvg()
+    {
+        $mockMimeTypeGuesserfactory = $this->createMock(MimeTypeGuesserFactoryInterface::class);
```
Comment (OskarStark) [inline src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php:18]: Do you really need a mock here?

```php
$factory = new MimeTypeGuesserFactory();
$handler = new ImageHandler(1, $factory,.....);
```
should be enuogh

`get()` will [return a new instance](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/master/src/Kunstmaan/MediaBundle/Helper/MimeTypeGuesserFactory.php) anyway 🤔

File: src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        $handler = new ImageHandler(1, $mockMimeTypeGuesserfactory, $mockExtensionGuesserfactory, 'aviaryKey');
+        $handler->setSlugifier(new Slugifier());
+
+        $media = new Media();
+        $media->setContent(new File(__DIR__.'/../../Fixtures/sample.svg'));
+
+        $handler->prepareMedia($media);
+
+        $this->assertSame(['original_width' => null, 'original_height' => null], $media->getMetadata());
```
Comment (OskarStark) [inline src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php:31]:
```suggestion
        self::assertSame(
            ['original_width' => null, 'original_height' => null],
            $media->getMetadata()
        );
```

File: src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php (hunk 1 above)
Comment (acrobat) [reply src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php]: Yes indeed, fixed!

File: src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php (hunk 2 above)
Comment (acrobat) [reply src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php:25]: I'm going to leave this for now, as we use the `$this->` "syntax" everywhere. So we can decided later what to use and php-cs-fixer to switch all usages at once.

**Comment by OskarStark** [comment]
Thanks for the feedback!

**Pull request merged**